src/olympia/reviewers/management/commands/auto_approve.py

```diff
@@ -188,6 +188,7 @@ def process(self, version_id):
             # has been rolled back, or they may not have been processed by the
             # on commit handler yet).
             _stop_queuing_tasks()
+            _discard_tasks()
```

Comment:
The comment above is no longer accurate with that addition and needs to be updated.